tracer/test/Datadog.Trace.Tools.dd_dotnet.ArtifactTests/CreatedumpTests.cs

```diff
@@ -68,20 +70,25 @@ public async Task Passthrough(string passthrough, bool shouldCallCreatedump)
         // "was COMPlus_DbgEnableMiniDump set?" check.
 
         SkipOn.Platform(SkipOn.PlatformValue.MacOs);
+        SkipOn.Platform(SkipOn.PlatformValue.Windows); // This test is not needed on Windows because we don't hook createdump
 
         using var reportFile = new TemporaryFile();
 
         (string, string)[] args = [LdPreloadConfig, .. CreatedumpConfig, ("DD_INTERNAL_CRASHTRACKING_PASSTHROUGH", passthrough), CrashReportConfig(reportFile)];
 
-        using var helper = await StartConsoleWithArgs("crash-datadog", false, args);
+        using var helper = await StartConsoleWithArgs("crash-datadog", true, args);
 
         await helper.Task;
 
         using var assertionScope = new AssertionScope();
         assertionScope.AddReportable("stdout", helper.StandardOutput);
         assertionScope.AddReportable("stderr", helper.ErrorOutput);
 
-        helper.StandardOutput.Should().Contain(CrashReportExpectedOutput);
+        if (RuntimeInformation.IsOSPlatform(OSPlatform.Linux))
```

Comment:
You already skip on os, so this is probably unneccessary

Reply:
Indeed